var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -65,7 +66,12 @@ class PyYt(PythonPackage):
     depends_on("py-sympy", type=('build', 'run'))
     depends_on("python @2.7:2.999,3.4:")
 
-    @run_after('install')
```

Comment:
@krafczyk Is selecting the phases at install-time a case that will be common in `PythonPackage`? I ask because in my original design for build-systems I assumed a static set of phases for a given build-system, and I am trying to figure out a solution for this.